docs/start/tutorial.md

```diff
@@ -496,14 +496,16 @@ export default function Contact() {
 
 TypeScript is very upset with us, let's make it happy and see what that forces us to consider:
 
-```tsx filename=app/routes/contacts.$contactId.tsx lines=[1,3,7-8]
-import type { LoaderArgs } from "@remix-run/node";
+```tsx filename=app/routes/contacts.$contactId.tsx lines=[1,3,7-10]
+import type { LoaderFunctionArgs } from "@remix-run/node";
```

Comment:
Ah good catch, I should have done a search for these when we did the renames the other week 👍 .  I'm going to do a separate quick PR (probably tomorrow AM) to find/replace this elsewhere in the docs.

Reply:
https://github.com/remix-run/remix/pull/7426